SimPEG/utils/mesh_utils.py

```diff
@@ -1,6 +1,7 @@
 import numpy as np
 
 from discretize.utils import (  # noqa: F401
+    random_model,
```

Comment:
Any reason to add the random model import from `discretize` here?

Reply:
I can't see why. I'm happy to remove it

Reply:
I would also consider moving it from discretize to SimPEG

Reply:
This can be done in a separate PR